Misc/NEWS.d/next/Windows/2021-04-08-00-36-37.bpo-34816.4Xe0id.rst

```diff
@@ -0,0 +1,2 @@
+``hasattr(ctypes.windll, 'nonexistant')`` now returns `False` instead of raising `exc:`OSError`.

```

Comment:
```suggestion
``hasattr(ctypes.windll, 'nonexistant')`` now returns ``False`` instead of raising :exc:`OSError`.

```

Reply:
Thanks! My dyslexia is really playing with me today and these ticks are small :rofl: